internal.nix

```diff
@@ -369,8 +369,8 @@ rec {
 
       in
       stdenv.mkDerivation ({
-        inherit (lockfile) version;
         pname = makeValidDrvName lockfile.name;
```

Comment:
Maybe this is a nitpick: but i wonder if it would be preferable to default to “no-version” instead so that there is a difference between a lockfile with version 0 and a lockfile that had no version where we defaulted to 0.

WDYT?